src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Button(props) {
```

Comment:
Nog beter zou zijn, als je het hier zou spreaden
```suggestion
function Button({ clickhandler, disabled, children }) {
```